diff --git a/01-archiso-x86_64-linux.conf b/01-archiso-x86_64-linux.conf
index d59262f..0a55b8e 100644
--- a/01-archiso-x86_64-linux.conf
+++ b/01-archiso-x86_64-linux.conf
@@ -1,6 +1,6 @@
-title   Arch Linux install medium (x86_64, UEFI)
+title   MagyArch Linux install medium (x86_64, UEFI)
sort-key 01
linux    /%INSTALL_DIR%/boot/x86_64/vmlinuz-linux
initrd   /%INSTALL_DIR%/boot/intel-ucode.img
initrd   /%INSTALL_DIR%/boot/amd-ucode.img
initrd   /%INSTALL_DIR%/boot/x86_64/initramfs-linux.img
-options  archisobasedir=%INSTALL_DIR% archisodevice=UUID=%ARCHISO_UUID%
+options archisobasedir=%INSTALL_DIR% cow_spacesize=4G archisolabel=%ARCHISO_LABEL%