diff --git a/packages.x86_64 b/packages.x86_64
index 28b2311..9db1d70 100644
--- a/packages.x86_64
+++ b/packages.x86_64
@@ -1,3 +1,47 @@
+# xorg
+xorg-drivers
+xorg-server
+xorg-xinit
+xorg-twm
+xterm
+
+lightdm
+lightdm-gtk-greeter
+
+# audio.
+alsa-utils
+
+# themes
+faenza-icon-theme
+
+
+# virtualbox
+# virtualbox-guest-utils
+# virtualbox-guest-modules-arch
+
+# cinemon
+cinnamon
+
+# egyéb
+base-devel
+# zenity
+reflector
+git
+firefox
+lxterminal
+leafpad
+bash-completion
+efitools
+gparted
+linux-headers
+rtl88x2bu-dkms-git
+
+# Installer
+yad-git
+gspell
+gtksourceview3
+
+
 alsa-utils
 amd-ucode
 arch-install-scripts
