diff --git a/profiledef.sh b/profiledef.sh
old mode 100644
new mode 100755
index 5feb205..6597f8a
--- a/profiledef.sh
+++ b/profiledef.sh
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 # shellcheck disable=SC2034

-iso_name="archlinux"
-iso_label="ARCH_$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y%m)"
-iso_publisher="Arch Linux <https://archlinux.org>"
-iso_application="Arch Linux Live/Rescue CD"
-iso_version="$(date --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y.%m.%d)"
+profile_name="MagyArchLinux"
+profile_label="MagyArch_Linux_x86_64"
+iso_name="MagyArchLinux"
+iso_label="MagyArch_Linux_x86_64"
+iso_publisher="MagyArch Linux <https://magyarchlinux.org>"
+iso_application="MagyArch Linux Live"
+iso_version=""
 install_dir="arch"
 buildmodes=('iso')
 bootmodes=('bios.syslinux.mbr' 'bios.syslinux.eltorito' 'uefi-x64.systemd-boot.esp' 'uefi-x64.systemd-boot.eltorito')
@@ -20,4 +20,9 @@ file_permissions=(
   ["/usr/local/bin/choose-mirror"]="0:0:755"
   ["/usr/local/bin/Installation_guide"]="0:0:755"
   ["/usr/local/bin/livecd-sound"]="0:0:755"
+  ["/usr/bin/beallit"]="0:0:755"
+  ["/usr/bin/Magyarcs"]="0:0:755"
+  ["/etc/skel/inditó.desktop"]="0:0:755"
+  ["/etc/skel/Desktop/inditó.desktop"]="0:0:755"
+  ["/indito"]="0:0:755"
 )
